docs/workflow/ci/pipelines-overview.md

```diff
@@ -0,0 +1,122 @@
+# Pipelines overview - Architecture and different available pipelines
+
+The runtime repository counts with a large amount of validation pipelines to help assess product quality accross different scenarios. Some of them run automatically, and some run per request to accommodate hardware availability and other resource constraints. However, the overall orchestration remains largely the same.
+
+```mermaid
+gitGraph
+    commit
+    commit
+    branch feature/utf8
+    checkout feature/utf8
+    commit
+    commit
+    checkout main
+    commit
+    merge feature/utf8 type: REVERSE
+    commit
+    commit
+```
+
+Say there's a PR from `feature/utf8` to `main`. The `Azure DevOps Pipeline` plugin will take the merge commit to `main` and queue all default pipelines and any other requested pipelines to `Azure DevOps`.
+
+```mermaid
+gantt
+    title Execution of a PR in our CI
+    dateFormat DDDD
+    axisFormat %j
+    section GH PR
+    Send PR using AZDO Plugin   : prCreate, 001, 1d
+    Workitem Analysis           : analysis, after lookup, 1d
+    Merge Step                  : after analysis, 1d
+    section Azure DevOps
+    Build Runtimes and Libs   : build, after prCreate, 1d
+    Build Tests               : buildTest, after build, 1d
+    section Helix
+    Run Tests                       : test, after buildTest, 1d
+    Report Tests to Azure DevOps    : testReport, after test, 1d
+    section Known Issues Infrastructure
+    Lookup known strings in issues  : lookup, after testReport, 1d
+```
+
+Each pipeline will create it's own build of the runtimes and the tests, and they eventually run them. We usually run our tests in a separate environment called Helix. This system allows for distribution of the large number of tests across the wide array of platforms supported. Once each worker machine processes its own results, these get reported back to `Azure DevOps` and they become available in the tests tab of the build.
+
+## Pipelines used in dotnet/runtime
+
+This repository contains several runtimes and a wide range of libraries supported in a wide array of platforms. This complexity makes it hard to balance resource usage, testing coverage, and developer productivity. In order to try to make build efforts more reliable and spend the least amount of time testing what the PR changes need we have various pipelines - some required, some optional. You can list the available pipelines by adding a comment like `/azp list` on a PR or get the available commands by adding a comment like `azp help`.
```

Comment:
Does `azp help` not need the heading `/` like `/azp list`?

Reply:
It does. I have largely copied these descriptions from prior docs. Thanks for rereading them.